src/core/textrenderer/qgstextrenderer.cpp

```diff
@@ -81,6 +81,7 @@ void QgsTextRenderer::drawText( const QRectF &rect, double rotation, Qgis::TextH
 
   // DO NOT USE _format in the following code, always use lFormat!!
 
+#if 0
```

Comment:
Is this #if 0 block lingering around for a reason or should it be taken out?

Reply:
Ah, should be removed. I'll kill it!